.github/workflows/develop-pipeline.yml

```
@@ -52,32 +53,140 @@
+        test-group: [
+          'repositories',
+          'adapters',
+          'resolvers',
+          'services',
+          'entities',
+          'server',
+          'utils',
+          'migrations'
+        ]
+    steps:
+      - uses: actions/checkout@v1
```

actionlint (1.7.7)

the runner of "actions/checkout@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)